ketcher-autotests/tests/Macromolecule-editor/RNA-Builder/rna-library.spec.ts

```diff
@@ -1378,4 +1383,165 @@ test.describe('RNA Library', () => {
 
     await page.getByTestId('cancel-btn').click();
   });
+
+  enum MonomerLocationTabs {
+    PEPTIDES = 'Peptides',
+    PRESETS = 'Presets',
+    SUGARS = 'Sugars',
+    BASES = 'Bases',
+    PHOSPHATES = 'Phosphates',
+    NUCLEOTIDES = 'Nucleotides',
+    CHEM = 'CHEM',
+  }
+
+  interface IIDTSearchString {
```

Comment:
In the `IIDTSearchString` interface, the `IDTSeatchString` property contains a typo ?